src/modules/score.js

```diff
@@ -0,0 +1,77 @@
+const form = document.querySelector('#add-new');
+const gameId = localStorage.getItem('gameId');
+
+const submitForm = async (event) => {
+  event.preventDefault();
+
+  const nameInput = document.getElementById('name');
+  const scoreInput = document.getElementById('score');
+
+  const data = {
+    score: scoreInput.value,
+    user: nameInput.value,
+  };
+
+  try {
+    if (!gameId) {
+      const createResponse = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          name: 'Soccer LeadorBoard',
+        }),
+      });
+
+      const createData = await createResponse.json();
+      const [, gameId] = createData.result.match(/ID: (\w+)/);
+      localStorage.setItem('gameId', gameId);
+    }
+
+    const scoresResponse = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+    await scoresResponse.json();
+
+    // alert('score posted successfully');
+  } catch (error) {
+    // alert('Something went wrong');
+  }
+};
+
+// Display data
+
+const dynamicDisplay = document.querySelector('#score-ul');
+const refreshButton = document.querySelector('#refresh');
+
+const refreshScores = async () => {
+  try {
+    const response = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores`);
+    const data = await response.json();
+    // eslint-disable-next-line no-alert
+    alert('Score received:', data);
+    // eslint-disable-next-line no-use-before-define
+    displayScores(data.result);
+  } catch (error) {
+    // eslint-disable-next-line no-alert
+    alert('Error fetching scores:', error);
+  }
```

Comment:
- [ ] It is advised that you do not make use of `alert` in your javascript code before it is not inline with best practices so I kindly suggest you remove these alert messages from your codebase 👍 
- [ ] Also kindly ensure you are not disabling linters for this project. 👍

Reply:
Fixed.